framework/test/unit/modules/token/cc_commands/cc_transfer.spec.ts

```diff
@@ -738,7 +738,7 @@ describe('CrossChain Transfer Command', () => {
 			});
 		});
 
-		it('should throw when the fee to initialize an account is insufficient', async () => {
+		it('should throw when escrow account has insufficient balance', async () => {
```

Comment:
I am a bit confused here. I don't see how balance of escrow connects with `execute` here.

Do you mean that the call of `initializeUserAccountInternal` fails (due to insufficient fee..)?

Reply:
There is an explicit check in [execute method](https://github.com/LiskHQ/lisk-sdk/blob/bd3212b0943570739021fedad921afbed55c8e3c/framework/src/modules/token/cc_commands/cc_transfer.ts#LL134C3-L134C3), this test targets that case.

Reply:
Ok I see. This check however does not appear in the LIP (see [here](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0051.md#execution-2)), since it is assumed to be checked already in verify. Does it make sense to keep this check here (cc. @shuse2)?

Reply:
I think we should keep the check for the amount to be safer